airbyte-integrations/connectors/source-salesforce/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: b117307c-14b6-41aa-9422-947e34922962
-  dockerImageTag: 2.4.4
+  dockerImageTag: 2.4.5
```

Comment:
should this be a major bump without a breaking change notice so it's clear it cannot be reverted?

Reply:
You are right! I'll update this